src/EnergyPlus/SizingManager.cc

```diff
@@ -1443,7 +1444,14 @@ void ManageSystemVentilationAdjustments(EnergyPlusData &state)
                     Real64 Er = TermUnitFinalZoneSizing(termUnitSizingIndex)
                                     .ZoneSecondaryRecirculation; // user input in Zone Air Distribution design spec object
 
-                    if (Er > 0.0) { // multi path zone
+                    if (state.dataSize->SysSizInput(SysSizNum).SystemOAMethod == SOAM_SP) { // 62.1 simplified procedure
+                        if (state.dataSize->DBySys(AirLoopNum) < 0.60) {
+                            state.dataSize->EvzByZoneHeat(termUnitSizingIndex) = 0.88 * state.dataSize->DBySys(AirLoopNum) + 0.22;
+                        } else {
+                            state.dataSize->EvzByZoneHeat(termUnitSizingIndex) = 0.75;
+                        }
+                        state.dataSize->EvzByZoneCool(termUnitSizingIndex) = state.dataSize->EvzByZoneHeat(termUnitSizingIndex);
+                    } else if (Er > 0.0) { // 62.1 ventilation rate procedure - multi path zone
```

Comment:
Calculation of the zone ventilation efficiency following the Standard 62.1 Simplified Procedure.